fdbclient/NativeAPI.actor.cpp

```diff
@@ -2356,15 +2356,24 @@ double Transaction::getBackoff(int errCode) {
 	return b;
 }
 
-void TransactionOptions::reset(Database const& cx) {
-	memset(this, 0, sizeof(*this));
-	maxBackoff = cx->transactionMaxBackoff;
-	lockAware = cx->lockAware;
+TransactionOptions::TransactionOptions(Database const& cx) {
+    maxBackoff = cx->transactionMaxBackoff;
```

Comment:
This is using spaces

Reply:
Done. I thought I'd set my local environment to use tabs, but alas, apparently not.